src/Standards/AcquiaPHP/ruleset.xml

```diff
@@ -102,131 +27,12 @@
-    <exclude name="Squiz.Arrays.ArrayDeclaration.KeySpecified"/>
-  </rule>
-  <!-- Disable some error messages that we do not want. -->
-  <rule ref="Squiz.Arrays.ArrayDeclaration.CloseBraceNotAligned">
-    <severity>0</severity>
-  </rule>
-  <rule ref="Squiz.Arrays.ArrayDeclaration.DoubleArrowNotAligned">
-    <severity>0</severity>
-  </rule>
-  <rule ref="Squiz.Arrays.ArrayDeclaration.FirstValueNoNewline">
-    <severity>0</severity>
-  </rule>
-  <rule ref="Squiz.Arrays.ArrayDeclaration.KeyNotAligned">
-    <severity>0</severity>
-  </rule>
-  <rule ref="Squiz.Arrays.ArrayDeclaration.MultiLineNotAllowed">
-    <severity>0</severity>
-  </rule>
-  <rule ref="Squiz.Arrays.ArrayDeclaration.NoComma">
-    <severity>0</severity>
-  </rule>
-  <rule ref="Squiz.Arrays.ArrayDeclaration.NoCommaAfterLast">
-    <severity>0</severity>
-  </rule>
-  <rule ref="Squiz.Arrays.ArrayDeclaration.NotLowerCase">
-    <severity>0</severity>
-  </rule>
-  <rule ref="Squiz.Arrays.ArrayDeclaration.SingleLineNotAllowed">
-    <severity>0</severity>
-  </rule>
-  <rule ref="Squiz.Arrays.ArrayDeclaration.ValueNotAligned">
-    <severity>0</severity>
-  </rule>
-  <rule ref="Squiz.Arrays.ArrayDeclaration.ValueNoNewline">
-    <severity>0</severity>
-  </rule>
-  <rule ref="Squiz.ControlStructures.ForEachLoopDeclaration"/>
-  <!-- Disable some error messages that we already cover. -->
-  <rule ref="Squiz.ControlStructures.ForEachLoopDeclaration.AsNotLower">
-    <severity>0</severity>
-  </rule>
-  <rule ref="Squiz.ControlStructures.ForEachLoopDeclaration.SpaceAfterOpen">
-    <severity>0</severity>
-  </rule>
-  <rule ref="Squiz.ControlStructures.ForEachLoopDeclaration.SpaceBeforeClose">
-    <severity>0</severity>
-  </rule>
-  <rule ref="Squiz.ControlStructures.ForLoopDeclaration"/>
-  <!-- Disable some error messages that we already cover. -->
-  <rule ref="Squiz.ControlStructures.ForLoopDeclaration.SpacingAfterOpen">
-    <severity>0</severity>
-  </rule>
-  <rule ref="Squiz.ControlStructures.ForLoopDeclaration.SpacingBeforeClose">
-    <severity>0</severity>
-  </rule>
-  <rule ref="Squiz.Functions.MultiLineFunctionDeclaration"/>
-  <rule ref="Squiz.Functions.MultiLineFunctionDeclaration.BraceOnSameLine">
-    <severity>0</severity>
-  </rule>
-  <rule ref="Squiz.Functions.MultiLineFunctionDeclaration.ContentAfterBrace">
-    <severity>0</severity>
-  </rule>
-  <!-- Standard yet to be finalized on this
-    (https://www.drupal.org/node/1539712). -->
-  <rule ref="Squiz.Functions.MultiLineFunctionDeclaration.FirstParamSpacing">
-    <severity>0</severity>
-  </rule>
-  <rule ref="Squiz.Functions.MultiLineFunctionDeclaration.Indent">
-    <severity>0</severity>
-  </rule>
-  <rule ref="Squiz.Functions.MultiLineFunctionDeclaration.CloseBracketLine">
-    <severity>0</severity>
-  </rule>
-  <rule ref="Squiz.Functions.FunctionDeclarationArgumentSpacing">
+  <!-- PSR-12 has no opinion on comments, leading to unpredictable indentation. -->
```

Comment:
> PSR-12 has no opinion on comments, leading to unpredictable indentation.

Therefore... ? I can't tell from looking at the name what `ignoreIndentationTokens = array` does, so it's not clear what this does to help.

Reply:
Sorry... specifically, the problem is this. When you upgrade from v2 to v3 and run cbf, literally every line of your project will change from 2-space indents to 4-space indents, _except_ inline comments, because PSR-12 explicitly ignores comment spacing/styling using the `ignoreIndentationTokens` array. So you get bastardized code like this:

```php
function foo() {
  // Call method bar().
    $this->bar();
}
```

By redeclaring this as an empty array, we persuade cbf to treat inline comments like all other code and indent them accordingly.

Reply:
Maybe something like this?

> Apply indentation rules to all kinds of code (i.e., tokens), including comments.